pkg/agent/controller/agent.go

```diff
@@ -219,34 +228,42 @@ func (a *Controller) Start(stopCh <-chan struct{}) error {
 	defer utilruntime.HandleCrash()
 
 	// Start the informer factories to begin populating the informer caches
-	logger.Info("Starting Agent controller")
+	agentLogger := logger.WithValues("ClusterID", a.clusterID)
+	agentLogger.Info("Starting Agent controller")
 
+	agentLogger.V(log.DEBUG).Info("Starting Agent controller")
```

Comment:
if verbosity==DEBUG won't this Info message be printed twice? Once in normal Info (line 232) and the other in line 234?